YOLayout/YOLayout_IMP.h

```diff
@@ -148,44 +138,17 @@ typedef enum {
 - (CGRect)setSize:(CGSize)size inRect:(CGRect)inRect view:(id)view options:(YOLayoutOptions)options;
 
 /*!
- Set origin.
+ Set origin. Same as setFrame:view: but uses the existing view.frame.size.
+
+ @param origin Origin
+ @param view View
  */
 - (CGRect)setOrigin:(CGPoint)origin view:(id)view;
```

Comment:
Do you use this at all? I think I'd be ok with removing it. Or making it `setOrigin:inRect:view:options:` to make it more generally useful